ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/DcatViewValidator.java

```diff
@@ -0,0 +1,67 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.validation;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.vocabulary.RDF;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import java.util.List;
+
+import static org.apache.jena.rdf.model.ResourceFactory.createProperty;
+
+@Component
+public class DcatViewValidator implements Validator {
+
+	public static final Property DCAT_DATA_SERVICE = createProperty("http://www.w3.org/ns/dcat#DataService");
+	public static final Property DCAT_DATASET = createProperty("http://www.w3.org/ns/dcat#Dataset");
+	public static final Property DCAT_CATALOG = createProperty("http://www.w3.org/ns/dcat#Catalog");
+	public static final Property DCAT_SERVES_DATASET = createProperty("http://www.w3.org/ns/dcat#servesDataset");
```

Comment:
you COULD put "http://www.w3.org/ns/dcat#" in a separate string and then call it in each of these statements. might read prettier

Reply:
Maybe one common place for everyone after all stories have been merged?